backend/hitas/views/apartment.py

```diff
@@ -615,3 +620,34 @@ def get_list_queryset(self):
             )
             .order_by("id")
         )
+
+    @action(detail=True, methods=["GET"])
+    def unconfirmed_prices_pdf(self, request, **kwargs) -> Union[HttpResponse, Response]:
+        apartment = self.get_object()
+        apartment_data = ApartmentDetailSerializer(apartment).data
+        ump = apartment_data["prices"]["maximum_prices"]["unconfirmed"]
+        ump = ump["pre_2011"] if ump["pre_2011"] is not None else ump["onwards_2011"]
+
+        if (
+            ump["construction_price_index"]["value"] is None
+            or ump["market_price_index"]["value"] is None
+            or ump["surface_area_price_ceiling"]["value"] is None
+        ):
+            return Response(
+                {
+                    "error": "index_missing",
+                    "message": "One or more indices required for max price calculation is missing.",
+                    "reason": "Conflict",
+                    "status": 409,
+                },
+                status=HTTPStatus.CONFLICT,
+            )
+
+        filename = f"Hinta-arvio {apartment.address}.pdf"
+        context = {"title": filename, "apartment": apartment_data}
+        pdf = render_to_pdf("unconfirmed_maximum_price.jinja", context)
+        response = HttpResponse(pdf, content_type="application/pdf")
+        # Download file for user instead of opening it in the browser
+        response.headers["Content-Disposition"] = f"attachment; filename={filename}"
+        response["Access-Control-Expose-Headers"] = "Content-Disposition"
```

Comment:
this should be set with `CORS_EXPOSE_HEADERS`? (https://github.com/adamchainz/django-cors-headers#cors_expose_headers-sequencestr)